project_template/models/project.py

```diff
@@ -1,11 +1,75 @@
 # © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
 
-from odoo import fields, models
+from odoo import api, fields, models
+from .project_task import SHOW_TASK_TEMPLATES
 
 
 class Project(models.Model):
 
     _inherit = 'project.project'
 
     is_template = fields.Boolean()
+
+    @api.model
+    def _create_task_from_template(self, template):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=4420733)